m3-sys/llvm/llvm9bindings/src/m3makefile

```diff
@@ -80,7 +83,7 @@ if equal (OS_TYPE, "POSIX")
 
   exec("make", "-f" , "../src/Makefile", "-k", "LLVM_INCLUDE_DIR=" & LLVM_INCLUDE_DIR, "LLVM_BUILD_INCLUDE_DIR=" & LLVM_BUILD_INCLUDE_DIR, "BUILD_DIR=" & BUILD_DIR)
 else
-  % Windows X86
+  % Both Windows x64 and Windows X86
```

Comment:
`amd64`

Reply:
It's fixed. See https://github.com/VictorMiasnikov/cm3/commit/b846c7e57e251066558c684f5c6c282d8bd23b91